scss/_patterns_accordion.scss

```diff
@@ -38,13 +33,17 @@
     z-index: 2;
 
     &::before {
```

Comment:
Since we seem to copy quite a bit of spacing from icon here, maybe `@extend %icon` would work instead?

Or at least we should have a comment in code mentioning that the vertical-align calculation is based on the one from `%icon`.

Reply:
Good point! I didn't realise I was repeating that many properties. Added `@extend %icon`